src/components/Paragraph/README.md

```diff
@@ -1,61 +1,105 @@
-## Available Sizes
-
-### Large Paragraph
+A `Paragraph` is a self-contained unit of one or more coherent sentences related to a single topic or idea.
 
 ```jsx
 <Paragraph size="large">
 	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
-	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	augue gravida hendrerit.
+</Paragraph>
+
+<Paragraph>
+	Vestibulum ornare eget lacus cursus rhoncus. Cras nec
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-### Medium Paragraph
+## Usage
+
+Use a `Paragraph` for any body copy comprising of one or more full sentences.
+
+Don't use a `Paragraph` for:
+- A single text link or call to action—use a [Button](../#/Function/Components/Button) instead.
+- A heading or subheading—use a `Heading`, `Subtitle`, or `Page Title`.
+- A list of related, equally weighted items—use a `List` instead.
+
+## Appearance
+
+- Line lengths should be between 55-70 characters.
+- Text colour should contrast with the background.
```

Comment:
You should mention the minimum acceptable contrast here if you have it handy ("WCAG AAA", etc.)

Reply:
Reading 'til the end: just a link to the `#accessibility` header should do. 😄